client.go

```diff
@@ -610,3 +611,17 @@ func (s *Client) LoopInTerms(ctx context.Context) (
 
 	return s.Server.GetLoopInTerms(ctx)
 }
+
+// wrapGrpcError wraps the non-nil error provided with a message providing
+// additional context, preserving the grpc code returned with the original
+// error. If the original error has no grpc code, then codes.Unknown is used.
+func wrapGrpcError(message string, err error) error {
```

Comment:
Is there an advantage of wrapping unknown errors in the few places in the PR vs just returning `status.Error` there with proper error codes?

Reply:
I think it's nice to be able to wrap with a bit more information and preserve the error? In the case of unknown errors, I think it's ok to add the unknown error code anyway, doesn't really take away much. We _could_ go through the whole api and add error codes everywhere, but I think surfacing the error codes returned by the server is a good start?